geode-lucene/build.gradle

```diff
@@ -71,6 +71,7 @@ dependencies {
 
   upgradeTestImplementation(project(':geode-dunit'))
   upgradeTestImplementation('commons-io:commons-io')
+  upgradeTestImplementation('org.apache.lucene:lucene-backward-codecs')
```

Comment:
just checking, is backward-codecs really only required by tests, not end-users too?  I would naively assume it needs to be shipped with Geode as well.

Reply:
I read some articles about Lucene and its similarities and only find that this backward-codecs is needed for test. So I'm not sure taht we need to ship it with Geode.